code/index.html

```diff
@@ -23,29 +26,282 @@
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jerry</p>
+          <img src="images/jerry.svg" alt="Jerry" />
+          <div class="guess">
+            <span>Guess on Jerry?</span>
+            <button class="filled-button small" onclick="guess('Jerry')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jess</p>
+          <img src="images/jess.svg" alt="Jess" />
+          <div class="guess">
+            <span>Guess on Jess?</span>
+            <button class="filled-button small" onclick="guess('Jess')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jocelyn</p>
+          <img src="images/jocelyn.svg" alt="Jocelyn" />
+          <div class="guess">
+            <span>Guess on Jocelyn?</span>
+            <button class="filled-button small" onclick="guess('jocelyn')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jon</p>
+          <img src="images/jon.svg" alt="Jon" />
+          <div class="guess">
+            <span>Guess on Jon?</span>
+            <button class="filled-button small" onclick="guess('Jon')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jordan</p>
+          <img src="images/jordan.svg" alt="Jordan" />
+          <div class="guess">
+            <span>Guess on Jordan?</span>
+            <button class="filled-button small" onclick="guess('jordan')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Josephine</p>
+          <img src="images/josephine.svg" alt="Josephine" />
+          <div class="guess">
+            <span>Guess on Josephine?</span>
+            <button class="filled-button small" onclick="guess('Josephine')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Josh</p>
+          <img src="images/josh.svg" alt="Josh" />
+          <div class="guess">
+            <span>Guess on Josh?</span>
+            <button class="filled-button small" onclick="guess('Josh')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Jude</p>
+          <img src="images/jude.svg" alt="Jude" />
+          <div class="guess">
+            <span>Guess on Jude?</span>
+            <button class="filled-button small" onclick="guess('Jude')">
+              Guess
+            </button>
+          </div>
+        </div>
+        <div class="card">
+          <p>Julie</p>
+          <img src="images/julie.svg" alt="Julie" />
+          <div class="guess">
+            <span>Guess on Julie?</span>
+            <button class="filled-button small" onclick="guess('Julie')">
```

Comment:
Is this part necessary? You have this part in the script that should cover this:
const generateBoard = () => {
  board.innerHTML = ''
  charactersInPlay.forEach((person) => {
    board.innerHTML += `
      <div class="card">
        <p>${person.name}</p>
        <img src=${person.img} alt=${person.name}>
        <div class="guess">
          <span>Guess on ${person.name}?</span>
          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
        </div>
      </div>
    `
  })
}